shared-libs/user-management/src/users.js

```diff
@@ -745,18 +746,17 @@ const getUserDoc = (username, dbName) => db[dbName]
     throw err;
   });
 
-const getUserDocsByName = (name) =>
-  Promise
-    .all(['users', 'medic'].map(dbName => getUserDoc(name, dbName)))
-    .catch(error => {
-      if (error.status !== 404) {
-        return Promise.reject(error);
-      }
-      return Promise.reject({
-        status: 404,
-        message: `Failed to find user with name [${name}] in the [${error.db}] database.`
-      });
+const getUserDocsByName = (name) => Promise
+  .all(['users', 'medic'].map(dbName => getUserDoc(name, dbName)))
+  .catch(error => {
+    if (error.status !== 404) {
+      return Promise.reject(error);
+    }
+    return Promise.reject({
+      status: 404,
+      message: `Failed to find user with name [${name}] in the [${error.db}] database.`
     });
+  });
```

Comment:
Maybe it's just me, but I think the two changes in this file should have braces and an explicit return. Basically if the entire function can't fit on one line I think it's clearer to have the old style. There are a bunch of places in this PR that have this style. What do you think?

Reply:
I agree with this!